experiments.go

```diff
@@ -8,4 +8,5 @@ type Experiments struct {
 	Libp2pStreamMounting bool
 	P2pHttpProxy         bool
 	StrategicProviding   bool
+	DHTCrawlerClient     bool
```

Comment:
Any ideas on a better name here? Looking for something more generic since the mechanisms underlying this new client may change.

Perhaps just a really generic `Experiments.NewDHTClient`?